fix the handle of thread and make following fragment work

lostwood opened the pull request:
-Create a threadpool in song and following fragment to work simultaneous
-Fix in the logic of following fragment to load the data
-Dimension the size of the image to fill in the imageview
-Refactor some conflict with the module in followin fragment

darkestpriest commented:
With this changes, the project cannot compile, please **always** check: compileJava, build and execute the app before make a pull-request. Thanks!

lostwood committed:
Fix:Change the adapter of following fragment

The convert of url to bitmap have to do it in the asyntask, because the http request need to be done inside the thread.

lostwood committed:
Feat:Extract Url from the description string

The url of the artist is not in bot object, so i have to pull the url from the description.

lostwood committed:
Fix:Change the handle of the thread

To work simultaneus the songfragment and the followingfragment the asynctask must be done in a threadpool

lostwood committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat

lostwood committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat

# Conflicts:
#	TKY/android/reference_wallet/fermat-tky-android-reference-wallet-fan-wallet-bitdubai/src/main/java/com/bitdubai/reference_wallet/fan_wallet/fragments/FollowingFragment.java

lostwood committed:
Refactor:Change the integration of following fragment and module

The way to call the module changed
